palindrome.c

```diff
@@ -0,0 +1,48 @@
+//Matthew Kellerman- sees if string is palindrome
```

Comment:
Perfect job! you got all the test cases perfectly (nice dealing with capitals - you're the first person so far to do so). Nice comments also!